### PR TITLE
[WIP]Stats API

### PR DESCRIPTION
**Pull request by rajivharlalka**
Update students and project pull+commit stats

### CODE REVIEW

File: utils/stats.go [hunk 1]
```diff
@@ -111,72 +111,107 @@ func FilterAndSaveCommits(API_URL string, LAST_COMMIT_SHA string) (bool, string)
...
+			for j := range files_arr {
+				file_map := files_arr[j].(map[string]interface{})
+				file_name := file_map["filename"].(string)
+				file_names = append(file_names, file_name)
+			}
+			languages_worked := GetLanguagesFromFilenames(file_names)
+			fmt.Println("languages worked is ", languages_worked)
+			// TODO: Update the Languages Worked Field under Student row
+			languages_worked=["abcd"]
+			db.Exec("UPDATE students SET tech_worked = ? WHERE username = ?", languages_worked, student_username)
```
Comment (rajivharlalka) [inline utils/stats.go:176]: is There any syntax error here , getting an error msg

image

Comment (rajivharlalka) [reply utils/stats.go:176]: Wrote 175 bcz there is some issue in find tech worked fucntion , and was always returing an empty array, so created an testing array just before updating

**Comment by sahil-shubham** [comment]
The github token I provided day expired, hope you aren't using that.

**Comment by rajivharlalka** [comment]
No, I used my own token

**Comment by rajivharlalka** [comment]
ISKI JARURAT NI

**Pull request closed without merging**